rules/PoiGeneric.js

```diff
@@ -164,8 +165,17 @@ function additiveScore(map, e1, e2) {
 
     var reason = result.reasons;
 
-    var searchRadius = Math.max(exports.getSearchRadius(e1), 
-        exports.getSearchRadius(e2));
+    var t1 = e1.getTags().toDict();
+    var t2 = e2.getTags().toDict();
+
+    var searchRadius = Math.max(exports.getSearchRadius(e1), exports.getSearchRadius(e2));
+
+    var stopPositionThreshold = 0.7;
+    if (getRelatedTagsByThreshold("public_transport=stop_position", e1.getTags().toDict(), stopPositionThreshold).length > 0 &&
+        getRelatedTagsByThreshold("public_transport=stop_position", e2.getTags().toDict(), stopPositionThreshold).length > 0)
+    {
+      searchRadius = Math.min(exports.getSearchRadius(e1), exports.getSearchRadius(e2));
```

Comment:
What I said isn't very clear.  More detail:

All of this is needed to catch the questionable bus stop reviews:

var stopPositionThreshold = 0.7;
    if (getRelatedTagsByThreshold("public_transport=stop_position", e1.getTags().toDict(), stopPositionThreshold).length > 0 &&
        getRelatedTagsByThreshold("public_transport=stop_position", e2.getTags().toDict(), stopPositionThreshold).length > 0)
    {
      searchRadius = Math.min(exports.getSearchRadius(e1), exports.getSearchRadius(e2));
    }

stopPositionThreshold was determined experimentally and was a way for me to be prevented from adding a hardcoded check for tag "highway=bus_stop" and "railway=tram_stop".  I figured that other public transportation stops should be treated in a similar fashion as those two.

However, without adding this:

{k:'railway', v:'tram_stop',              match:250,      review:500},

I wasn't able to remove the questionable tram stop reviews.  That was what I was referring to above as being a little confusing to me.